packages/template-retail-react-app/app/components/social-login/index.jsx

```diff
@@ -19,14 +19,21 @@ import {AppleIcon, GoogleIcon} from '@salesforce/retail-react-app/app/components
  * @returns
  */
 const SocialLogin = ({idps}) => {
+    const intl = useIntl()
     const IDP_CONFIG = {
```

Comment:
@hajinsuha1 We keep the IDP_CONFIG here and the `idps` as a prop. What happens if idps=['instagrame'] and IDP_CONFIG is missing instagram object. As a result this will render an empty. Should we check that the config has what is being mentioned in idps props before rendering?

Reply:
Great point. Currently the code will just "do nothing" (return null and not render the button) when an idp is missing from IDP_CONFIG.

I've added a logger.error to make the failure more loud!